### ISSUE TITLE
On some versions of MacOS you need to #include <sys/ucontext.h> instead of #include <ucontext.h>

### ISSUE THREAD

**Issue by catolm** [xoreaxeaxeax/sandsifter#20]
On some versions of MacOS you need to #include <sys/ucontext.h> instead of #include <ucontext.h>

**Comment by Lewiscowles1986** [comment]
Do you have more information on pre-processor checks that can be used?

**Comment by sinetek** [comment]
can you compile at all? i'm getting a lot of errors on 10.12

**Comment by liuziangexit** [comment]
@catolm excuse me but why? 

Whats the difference between ucontext.h and sys/ucontext.h?

danke!

**Comment by philmb3487** [comment]
probably just the symbolic link..

sre, 11. dec 2019. 14:27 liuziangexit <notifications@github.com> je
napisao/la:

**Comment by liuziangexit** [comment]
yeah i noticed that in some newer version of macOS(like 10.14), the ucontext won't work... it will gives you unexpected segfault. i have no idea with this.
